src/frontend/components/style.scss

```diff
@@ -0,0 +1,8 @@
+$blinkas-red: #ff0000;
```

Comment:
This is just a repetition of styles you have created in another PR. If there happen to be dependencies in your work, kindly prompt your instructor for help on it.